config/services/_main.yaml

```diff
@@ -10,9 +10,22 @@ parameters:
 services:
     # default configuration for services in *this* file
     _defaults:
-        autoconfigure: true
+        # Automatically injects dependencies in your services
         autowire: true
 
+        # Automatically registers your services as commands, event subscribers, etc.
+        autoconfigure: true
+
+        # Allows optimizing the container by removing unused services; this also means
+        # fetching services directly from the container via $container->get() won't work
+        public: false
+
+    _instanceof:
```

Comment:
Why do you disable autowiring for these classes ?

Reply:
Cause it's not possible to autowire them and resource bundle do the job. Same pull request on Sylius standard has been merged https://github.com/Sylius/Sylius-Standard/pull/373

Reply:
Here is some description of the issue: https://forum.sylius.com/t/how-to-create-my-own-custom-resource-with-custom-controller-without-getting-can-not-autowire-exception/1292?u=lchrusciel